packages/cashier/src/pages/account-transfer/account-transfer-receipt/account-transfer-receipt.tsx

```diff
@@ -156,7 +155,17 @@ const AccountTransferReceipt = observer(({ onClose, history }: TAccountTransferR
                     />
                 </Modal.Body>
                 <Modal.Footer>
-                    <Button has_effect text={localize('Cancel')} onClick={toggleSwitchAlert} secondary large />
+                    <Button
+                        has_effect
+                        text={localize('Cancel')}
+                        onClick={() => {
+                            toggleSwitchAlert();
+                            setShouldSwitchAccount(false);
+                            onClose?.();
```

Comment:
can you explain the reason behind this line?
Shouldn't this `onClose()` be enough?
```suggestion
                            onClose();
```

Reply:
Already resolved 😄 👍🏼